src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/module.js

```diff
@@ -53,7 +53,7 @@ ts.ui.FormsModule = gui.module('forms-gui@tradeshift.com', {
 				['.ts-form input[type=date]', ts.ui.DateInputSpirit],
 				['.ts-form span + input[type=checkbox]', ts.ui.SwitchSpirit],
 				['.ts-form input[type=checkbox], .ts-form input[type=radio]', ts.ui.OptionSpirit],
-				['.ts-form input', ts.ui.TextInputSpirit],
+				['.ts-form span + input', ts.ui.TextInputSpirit],
```

Comment:
I guess this could break some form with no label; and we have some of those around :cry: